narwhal/primary/src/state_handler.rs

```diff
@@ -129,7 +133,7 @@ impl StateHandler {
             .map(|info| info.name)
             .collect();
 
-        self.network.unreliable_broadcast(our_workers, &message);
+        self.network.broadcast(our_workers, &message)
```

Comment:
Should this change to a retrying message be done here or offered in a new `reliably_notify_our_workers` method?

Reply:
To be honest I tried to make the minimum amount of work here as we are going to remove this call between primary -> worker anyways . Is the `reliably_notify_our_workers` method you suggest part of the state handler or make it part of the network?